libr/core/cmd_anal.c

```diff
@@ -720,7 +720,7 @@ static const char *help_msg_ax[] = {
 	"axj", "", "list refs in json format",
 	"axF", " [flg-glob]", "find data/code references of flags",
 	"axm", " addr [at]", "copy data/code references pointing to addr to also point to curseek (or at)",
-	"axt", " [addr]", "find data/code references to this address",
+	"axt", "[?] [addr]", "find data/code references to this address",
```

Comment:
this `?` suffix should work on all the commands

Reply:
It will once we fully switch to newshell.